server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/CassandraBlobStoreTest.java

```diff
@@ -102,6 +103,12 @@ public void deleteBucketShouldThrowWhenNullBucketName() {
 
     }
 
+    @Override
+    @Disabled("Concurrent read and delete can lead to partial reads (no transactions)")
```

Comment:
Are you sure it can't be done?

Reply:
Can be done: I was looking at the wrong root-cause. The root cause is the fact that PipedStream do not propagate exception. Extending them allows to solve this.